criu/kerndat.c

```diff
@@ -341,6 +341,7 @@ static int kerndat_get_dirty_track(void)
 
 	map[0] = '\0';
 
+	/* coverity[check_return] */
 	lseek(pm2, (unsigned long)map / PAGE_SIZE * sizeof(u64), SEEK_SET);
```

Comment:
Here we probably need to check a return code or we need to add a comment why we don't need to do this here

Reply:
If I check the return code, CRIU fails with:
```
+ ./criu/criu check
Error (criu/kerndat.c:346): Seek pmap err!: No such file or directory
Error (criu/kerndat.c:1084): kerndat_get_dirty_track failed when initializing kerndat.
Error (criu/crtools.c:213): Could not initialize kernel features detection.
```
Not sure why we do it if it fails.